templates/e-commerce/api/high-low-all-products.json

```diff
@@ -1,88 +1,88 @@
 {
```

Comment:
FYI all the json files in this folder are generated by the build.js script. They've been updated to have more varying placeholder text.